cocotb/monitors/__init__.py

```diff
@@ -54,16 +54,28 @@ def __init__(self):
 
 
 class Monitor(object):
+    """Base class for Monitor objects. Monitors are passive 'listening' objects
```

Comment:
Nice docstring, I just don't understand enough of the internals to review the content. :)